config/authorino/rbac/authconfig_viewer_role.yaml

```diff
@@ -0,0 +1,20 @@
+# permissions for end users to view services.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: authconfig-viewer-role
```

Comment:
We need to do this change in http://github.com/kuadrant/authorino – i.e. fix the name of the file and name of the resource.

Should we be committing these `config/authorino/rbac/*` files if Authorino is installed by git cloning the repo and installing from the manifests hosted there?

Reply:
About committing these files, having all the manifests (Operator's and Authorino's) in one single place – actually even in one single multi-document YAML file – can be useful to simplify installation. Instead of cloning the repo, one could install by applying directly sth like https://raw.githubusercontent.com/kuadrant/authorino-operator/main/config/manifests.yaml, plus deploying the operator from a pre-built public image.

Reply:
Authorino's CRD and RBAC will be installed via the operator by applying the manifests available on the authorino's repo once this [issue](https://github.com/Kuadrant/authorino/issues/190) is resolved